docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -88,62 +84,93 @@ To enable your application to use the image util functionality:
    ret = image_util_decode_create(&decode_h);
    ret = image_util_decode_set_input_path(decode_h, SAMPLE_FILENAME);
    ret = image_util_decode_set_colorspace(decode_h, colorspace);
-   ret = image_util_decode_set_output_buffer(decode_h, &img_source);
 
-   ret = image_util_decode_run(decode_h, &width, &height, &size_decode);
+   ret = image_util_decode_run2(decode_h, &src_image);
    ret = image_util_decode_destroy(decode_h);
+   ```
 
-   ret = media_format_create(&fmt);
-   ret = media_format_set_video_mime(fmt, colorspace);
-   ret = media_format_set_video_width(fmt, width);
-   ret = media_format_set_video_height(fmt, height);
+<a name="colorspace"></a>
+## Converting the Color Space of an Image
 
-   ret = media_packet_create_alloc(fmt, NULL, NULL, &src);
-   ret = media_packet_get_buffer_size(src, &size);
+To convert an image from one color space of an image to another:
 
-   src = malloc(size);
-   ret = media_packet_get_buffer_data_ptr(src, &src_ptr);
-   ret = memcpy(src_ptr, img_source, size);
+1. Create a transformation handle using the `image_util_transform_create()` function:
```

Comment:
Thanks for your review.
I will remove "the" and "function" in case of "the *** function".
I have one question. Should I remove "the" and "callback" in case of "the *** callback"?